src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+}
+
+.logo {
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  padding: 10px 50px;
+}
+
+.nav {
+  text-align: right;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  list-style: none;
+  margin: auto 40px;
+}
+
+.nav__item {
+  display: inline-block;
+  height: 60px;
+}
+
+.nav__link {
+  display: inline-block;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  transition: color 0.3s;
+  padding: 22.5px 0;
+  margin: 0 9px;
```

Comment:
Set margins only from one side, for example margin-right, but not apply it for last link
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/220726108-ebc3fd06-1f3c-448b-ab50-b69868b7a102.png">